presto-main/src/main/java/com/facebook/presto/sql/rewrite/ShowQueriesRewrite.java

```diff
@@ -477,6 +487,41 @@ private static String getFunctionType(SqlFunction function)
         }
 
         @Override
+        protected Node visitShowColumnStats(ShowColumnStats node, Void context)
+        {
+            QualifiedName table = node.getTable();
+            TableStatistics tableStatistics = getTableStatistics(node, table);
+
+            Set<String> statisticsNames = findUniqueStatisticsNames(tableStatistics);
+
+            List<String> resultColumnNames = buildColumnsNames(statisticsNames);
+            SelectItem[] selectItems = buildSelectItems(resultColumnNames);
```

Comment:
make buildSelectItems return `List<SelectItem>` and convert to array inline when passing to selectList method.
Or just use `new Select(false, List)`